libraries/session-storage/impl/src/main/sqldelight/migrations/5.sqm

```diff
@@ -0,0 +1,3 @@
+-- Migrate DB from version 5
```

Comment:
Maybe document here that for migrating users, we do not force the verification of the session?

Reply:
After internal discussion, I am not sure about the choice to let existing unverified user access the application. Maybe just force users to verify their session, even after an upgrade of the application?
But the value ("is the session verified") would have to be read from the SDK.
CC @pmaier1